File: mobile_client/app/src/main/java/wpam/mobile_client/protocol/StopStreamBuilder.java
package wpam.mobile_client.protocol;

import java.io.Closeable;
import java.util.ArrayList;

	public final class StopStreamBuilder implements CommandBuilder, Closeable
	{
		public StopStreamBuilder()
		{
		}
		public final void close()
		{
		}

		@Override
		public Command create(ArrayList<Integer> commandInBytes)
		{
			Command command  = new StopStreamCommand();
			return command;
		}
	}

File: server/src/protocol/StopStreamBuilder.cpp
#include "StopStreamBuilder.h"
#include "StopStreamCommand.h"

using namespace std;
using namespace communication;

StopStreamBuilder::StopStreamBuilder()
{}

StopStreamBuilder::~StopStreamBuilder()
{}

unique_ptr<Command> StopStreamBuilder::create(const vector<uint8_t> &commandInBytes)
{
    auto command = make_unique<StopStreamCommand>();

    return move(command);
}